framework/doc/content/source/constraints/EqualValueEmbededConstraint.md

```diff
@@ -0,0 +1,49 @@
+# EqualValueEmbededConstraint
+
+This is a constraint acting upon overlapping portions of two blocks, a slave block and a master block. The constraint enforces the slave variable on the slave block and the master variable on the master block to have the same values. Mesh dimensions of two blocks do not have to match.
+
+The constraint iterates through all the slave nodes on the slave block and finds associated master element. If a slave node has a master element, then contact force is computed according to user-specified contact model and formulation.
+
+## Mathematical Introduction
+
+The constraint utilizes contact models. Currently supported models and formulations are listed below.
+
```

Comment:
Since you got rid of the "model" option, remove the text below about "Model: Glued". Also, I would significantly re-work this.  Here's a proposal:
```
## Mathematical Formulation
Options are available to control how this constraint is applied:

### Kinematic Formulation
This option strictly enforces value of the solution at the slave nodes to be equal to the value in the master element at that point. The constraint is enforced by updating the slave residual $r_s$ and master residual $r_m$ as:
\begin{equation}
\begin{aligned}
r_s &= r_s + f_c + k_p(u_{slave} - u_{master})\\
r_m &= r_m + \phi_i r_{s,copy}
\end{aligned}
\end{equation}
where $r_{s,copy}$ is a copy of the residual of the slave node before the constraint is applied and $k_p$ is the user-specified penalty parameter. This formulation uses the penalty parameter only to penalize the error, and the converged solution has no error due the penalty. The penalty factor must be specified, and should be consistent with the scaling for the solution variable to which this is applied.

## Penalty Formulation
This option uses a penalty formulation in which the error in the solution is proportional to a user-specified penalty parameter $k_p$. The constraint is enforced by modifying the slave and master residual 
The constraint is enforced by updating the slave residual $r_s$ and master residual $r_m$ as:
\begin{equation}
\begin{aligned}
r_s &= r_s + k_p(u_{slave} - u_{master})\\
r_m &= r_m - \phi_i k_p(u_{slave} - u_{master})
\end{aligned}
\end{equation}
where $r_{s,copy}$ is the ghosted residual. The penalty parameter must be selected carefully, as small values lead to large differences between the slave node's solution and the solution in the master element, while large values may lead to poor convergence.